dart/lib/src/protocol/sentry_stack_trace.dart

```diff
@@ -25,6 +27,23 @@ class SentryStackTrace {
   /// thus mapping to the last frame in the list.
   Map<String, String> get registers => Map.unmodifiable(_registers ?? const {});
 
+  /// The language of the stacktrace
+  final String? lang;
+
+  /// Indicates that this stack trace is a snapshot triggered
+  /// by an external signal.
+  ///
+  /// If this field is false, then the stack trace points to the code that
+  /// caused this stack trace to be created.
+  /// This can be the location of a raised exception, as well as an exception or
+  /// signal handler.
+  ///
+  /// If this field is true, then the stack trace was captured as part
+  /// of creating an unrelated event. For example, a thread other than the
+  /// crashing thread, or a stack trace computed as a result of an external kill
+  /// signal.
+  final bool? snapshot;
```

Comment:
I'm wondering whether these should be set for native android stacktraces which are reported through PlatformExceptions

Reply:
Likely yes, great attention to detail.

Reply:
I already have a follow-up for this one, but I can't open a PR which has this branch as a target, so I'll open it after this has landed.

Reply:
Does it need to be set somewhere in the Dart code, too?

Reply:
No, I don't think so.